homeassistant/components/sonarr/manifest.json

```diff
@@ -3,9 +3,9 @@
   "name": "Sonarr",
   "documentation": "https://www.home-assistant.io/integrations/sonarr",
   "codeowners": ["@ctalkington"],
-  "requirements": ["sonarr==0.3.0"],
+  "requirements": ["aiopyarr==22.2.1"],
   "config_flow": true,
   "quality_scale": "silver",
   "iot_class": "local_polling",
-  "loggers": ["sonarr"]
+  "loggers": ["aiopyarr"]
```

Comment:
@bdraco how do we handle loggers for integrations that share a common package, ie radarr and lidarr both will use this aiopyarr as well

Reply:
It just means that the logging can include extra info from the other integration. There is nothing we can do unless the library solves it.

Reply:
This way the user can see the air time of the episode, not just the date.